src/exchange/group/accounts/group-accounts.controller.js

```diff
@@ -1,7 +1,8 @@
 angular.module("Module.exchange.controllers")
     .controller("ExchangeGroupAccountsCtrl", class ExchangeGroupAccountsCtrl {
-        constructor ($scope, Exchange, messaging, navigation, translator) {
+        constructor ($rootScope, $scope, Exchange, messaging, navigation, translator) {
             this.services = {
+                $rootScope,
```

Comment:
Why this.services.$rootScope -> this.$rootScope, no ?
Same for $scope

Reply:
It's kind of a good practice to differenciate services and actual controller properties, which is something done in Angular 2+, but as I'm the only one to do it I guess I'll have to let it go

https://www.youtube.com/watch?v=moSFlvxnbgk